downstream/modules/platform/proc-contoller-ee-mount-options.adoc

```diff
@@ -0,0 +1,46 @@
+[id="proc-controller-ee-mount-options"]
+
+= Execution environment mount options
+
+Rebuilding an is one way to add certificates, but inheriting certificatess from the host provides a more convenient solution. 
+For VM-based installs, {ControllerName} automatically mounts the system trust store in the when jobs run.
+
+Additionally, you may customize mount options and mount paths in the *Paths to expose to isolated jobs* field of the *Job Settings* page, where it supports podman-style volume mount syntax. 
+
+For more information, see the link:https://docs.podman.io/en/latest/markdown/podman-run.1.html#volume-v-source-volume-host-dir-container-dir-options[Podman documentation].
+
+In some cases where the `/etc/ssh/*` files were added to the image due to customization of an {ExecEnvShort}, an SSH error can occur. 
+For example, exposing the `/etc/ssh/ssh_config.d:/etc/ssh/ssh_config.d:O` path enables the container to be mounted, but the ownership permissions are not mapped correctly.
+
+Use the following procedure tf you encounter this error, or have upgraded from an older version of {ControllerName}:
+
+.Procedure
+. Change the container ownership on the mounted volume to `root`.
+. In the *Paths to expose to isolated jobs* field of the Job Settings page, using the current example, expose the path as such:
++
+image:settings-paths2expose-iso-jobs.png[Paths]]
++
+[NOTE]
+====
+The `:O` option is only supported for directories. 
+You must be as specific as possible, especially when specifying system paths. 
```

Comment:
Might be just me and my brain been fried from controller but this sentence seems a mouthful, suggested re-write:

"Be as detailed as you can, especially when specifying system paths."